ethdrain.py

```diff
@@ -131,18 +132,39 @@ def setup_process(block_range):
 
 
 if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument('-s', '--start',
+                        default='0',
+                        help='What block to start indexing'
+                        )
+
+    parser.add_argument('-e', '--end',
+                        default='1000',
+                        help='What block to finish indexing'
+                        )
+
+    parser.add_argument('-u', '--esurl',
+                        default='http://localhost:9200',
+                        help='The elasticsearch url and port. Accepts all the same parameters needed as a normal Elasticsearch client expects.'
+                        )
+
+    parser.add_argument('-m', '--esmaxsize',
+                        default='10',
+                        help='The elasticsearch max chunk size'
+                        )
+
+    args = parser.parse_args()
 
-    if len(sys.argv) == 2:
-        with open(sys.argv[1]) as f:
-            CONTENT = f.readlines()
-            BLOCKS_TO_PROCESS = [int(x) for x in CONTENT if x.strip() and len(x.strip()) <= 8]
-    elif len(sys.argv) == 3:
-        BLOCKS_TO_PROCESS = list(range(int(sys.argv[1]), int(sys.argv[2])))
-    else:
-        sys.exit("Usage: ethdrain.py <block start> <block end> OR <block list file>")
+    esurl = args.esurl
 
-    CHUNKS = list(chunks(BLOCKS_TO_PROCESS, CHUNK_SIZE))
+    esmaxsize = int(args.esmaxsize) | CHUNK_SIZE
```

Comment:
Unsure about that one: esmaxsize is not the same as CHUNK_SIZE. And since it's default initialized by argparse, CHUNK_SIZE will never be used.
Is that expected?